web/server/codechecker_server/database/run_db_model.py

```diff
@@ -456,6 +456,35 @@ def __init__(self, name, value, description=None, user_name=None):
         self.username = user_name
 
 
+class CleanupPlan(Base):
+    __tablename__ = 'cleanup_plans'
+
+    name = Column(String, nullable=False, primary_key=True)
+    due_date = Column(DateTime, nullable=True)
+    description = Column(String, nullable=True)
+    closed_at = Column(DateTime, nullable=True)
+
+    def __init__(self, name, due_date=None, description=None, closed_at=None):
+        self.name = name
+        self.due_date = due_date
+        self.description = description
+        self.closed_at = closed_at
+
+
+class CleanupPlanReportHash(Base):
+    __tablename__ = 'cleanup_plan_report_hashes'
+
+    cleanup_plan_name = Column(
+        Integer,
+        ForeignKey('cleanup_plans.name',
```

Comment:
Is it worth using a string as primary key? I mean using this column as foreign key for joining a table may unnecessarily increase database size. Probably this is not a huge date compared to other tables, still this could be some easy save.

Reply:
I changed the database schema to use an id primary column. I also refactored the API functions to use id instead of name parameter.